src/dev/flang/be/jvm/JVM.java

```diff
@@ -600,6 +645,8 @@ void finish(JVM jvm)
 
   Runner _runner;
 
+  JarOutputStream _jos;
+
```

Comment:
Please add a comment what this is (same for my code where these comments are missing, but in a separeate patch).

Reply:
Will add a PR for `_numLocals` documentation, that is really very cryptic.